pages/burials-and-memorials/application/530.md

```diff
@@ -1,17 +0,0 @@
----
-title: Apply for Burial Benefits (VA Form 21P-530)
-display_title: Apply Now
-description: Use VA Form 21P-530 to apply for VA burial benefits online now. Get step-by-step instructions, and consider signing in to save your in-progress application if you need to come back to finish filling it out later.
-entryname: burials
-layout: page-react.html
-order: 1
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/burials-memorials/"> Burials and Memorials </a></li>
-    <li><a href="/burials-memorials/dependency-indemnity-compensation/"> Survivors and Dependents </a></li>
-    <li><a aria-current="page" href="/burials-memorials/veterans-burial-allowance/">Handling Burial Costs (Allowances)</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-bc54ae76a6add12a02119b1a3de6a07cR5.